### ISSUE TITLE
Make it possible to translate the installer

### ISSUE THREAD

**Issue by ppvnf** [martinet101/WingetUI#260]
I guess this would apply to ElevenClock as well

**Comment by ppvnf** [comment]
@martinet101 Is it gonna be possible to also use Tolgee to translate it?

**Comment by martinet101** [comment]
The installer is inno, I have added the official language packs but I'll search for some other. There are like 20 different languages

**Comment by ppvnf** [comment]
> The installer is inno, I have added the official language packs but I'll search for some other. There are like 20 different languages

Is custom text like [this](https://github.com/martinet101/WingetUI/pull/313/commits/f0dd234129f6f5dfdf441fbe1595139b894e4fc1) gonna be translated?

**Comment by martinet101** [comment]
No, but honestly, those are two text strings that users will face only one time (and zero if they happen to install WingetUI through winget or scoop), and making an engine on inno is a complete mess and, in my opinion, too much work and effort for a single, one-time thing that might even not show at all.

I guess you understand the reasons of this decision

**Comment by ppvnf** [comment]
You're probably right. I guess most users only see the next button and click on it until the install is done and more advanced users know at least a little bit of English 🤣
